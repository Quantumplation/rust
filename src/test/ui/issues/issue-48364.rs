fn foo() -> bool {
    b"".starts_with(stringify!(foo))
    //~^ ERROR arguments to this function are incorrect
}

fn main() {}
