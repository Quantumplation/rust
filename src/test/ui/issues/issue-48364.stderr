error[E0308]: arguments to this function are incorrect
  --> $DIR/issue-48364.rs:2:9
   |
LL |     b"".starts_with(stringify!(foo))
   |         ^^^^^^^^^^^ --------------- expected `&[u8]`, found `&'static str`
   |
help: provide an argument of the correct type
   |
LL |     b"".({&[u8]})(stringify!(foo))
   |         ^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
